.github/workflows/e2e.yml

```diff
@@ -683,10 +676,27 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ always() && contains(github.ref, 'k8s-sync') }}
     steps:
-      - name: notification
+      - name: Get vcluster commit url
+        run: |
+          VCLUSTER_REF=$(echo ${{ github.event.head_commit.message }} | awk '{print $NF}')
+          echo "VCLUSTER_URL=${{ github.server_url }}/vertica/vcluster/commit/$VCLUSTER_REF" >> $GITHUB_ENV
+
+      - name: notification on success
+        if: ${{ always() && contains(needs.sync-vcluster.result, 'success') }}
+        uses: jdcargile/ms-teams-notification@v1.4
+        with:
+          github-token: ${{ github.token }}
+          ms-teams-webhook-uri: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
+          notification-summary: sync-vcluster job in worflow ${{ github.workflow }} with vcluster commit ${{ evn.VCLUSTER_URL }} on branch ${{ github.ref }} finished with status: ${{ needs.sync-vcluster.result }}
```

Comment:
This message cannot show correctly in the Teams channel. What is "evn" here? Should it be "env"?

Reply:
Updated the typo together with the format. Now the vcluster commit can be viewed as a link now.